frontend/src/utils/theme-changer.ts

```diff
@@ -1,10 +1,6 @@
-import { useLayoutEffect } from 'react';
-
 const useThemeClass = () => {
-  useLayoutEffect(() => {
-    const storedTheme = localStorage.getItem('theme');
-    storedTheme === 'dark' && document.documentElement.classList.add('dark');
-  }, []);
+  const storedTheme = localStorage.getItem('theme');
+  storedTheme === 'dark' && document.documentElement.classList.add('dark');
 };
```

Comment:
So now lets say i have dark mode enabled in my browser and then i visit the website for the first time. I don't think so that time it will work right? because it is just taking from the localstorage.... and this is done in the 2nd iteration, but the 1st walaa still we are missing. I guess we have to import some code from theme-toggle-button and port it here, 

And in that button, we just change the localstorage and the actual  theme right? 

Let's think a bit broader here. and try to figure out all the corner case

Reply:
Ok sir will try to figure it out